betfairlightweight/streaming/cache.py

```diff
@@ -508,7 +504,7 @@ def serialise(
             "lapseStatusReasonCode": self.lapse_status_reason_code,
             "cancelledDate": self._cancelled_date_string,
         }
-        if lightweight is False:  # cache resource
+        if not lightweight:  # cache resource
```

Comment:
Is there a chance lightweight could be None at this point which is equivalent to False?

Reply:
Someone might feed it in, but if that's intended, then it shouldn't be type hinted as `bool`.